config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-upgrade.yaml

```diff
@@ -0,0 +1,30 @@
+# periodic jobs
+
+periodics:
+- name: ci-kubernetes-e2e-kubeadm-kinder-upgrade-stable-master
+  interval: 1h
+  decorate: true
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  decoration_config:
+    timeout: 2400000000000 #40m
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  path_alias: k8s.io/kubeadm
```

Comment:
i have a quick question here.
if i remove this line where would my `$(pwd)` be?
i.e. how would that affect:
```
      - "./kinder/ci/kinder-run.sh"
```
i tried finding the documentation for that, but i couldn't .

Reply:
The docs are here, I can't say I actually grok them enough to answer your question though: https://github.com/kubernetes/test-infra/blob/master/prow/pod-utilities.md

Relevant lines seem to be 

>`periodic` jobs will have the working directory set to the root of the repo specified by the first ref in `extra_refs` (if specified)

>Repos requiring a non-standard clone path can use the `path_alias` field
to clone the repo to different go import path than the default of `/home/prow/go/src/github.com/{{.Org}}/{{.Repo}}/` (e.g. `path_alias: k8s.io/test-infra` -> `/home/prow/go/src/k8s.io/test-infra`)

Reply:
thanks, i read that page but missed the part about the first extra_ref!